python/skupper_router_internal/dispatch.py

```diff
@@ -78,7 +76,7 @@ class IoAdapter:
     def __init__(self, handler: Callable, address: str, aclass: str, treatment: int) -> None:
         ...
 
-    def send(self, message: 'Message', no_echo: int, control: int) -> None:
+    def send(self, message: 'Message', no_echo: bool = True, control: bool = False) -> None:
```

Comment:
What is the difference between `no_echo: bool = True` and just `no_echo=True` ? Is the latter old school or does the latter not fix the linter error ?

Reply:
I didn't want to change the original meaning of the arguments - doing so would involve finding all the callers and changing them to adopt the new meaning. Beyond the scope of this fix.

Reply:
The bool designation is used by the linters as a hint to check the callers to ensure they are invoking the method with the proper types.

'typing' (type hints) is a new(ish) feature in python and helps static analysis - we really should use it going forward.  See https://docs.python.org/3/library/typing.html?highlight=typing#module-typing

Reply:
@kgiusti It would help if you showed what linter warnings you are actually fixing here

Reply:
@jiridanek ah, yes - sorry about that:



    67: ************* Module skupper_router_internal.dispatch
    67: /home/kgiusti/work/skupper/skupper-router/python/skupper_router_internal/dispatch.py:44:0: W0404: Reimport 'ctypes' (imported line 38) (reimported)
    67: /home/kgiusti/work/skupper/skupper-router/python/skupper_router_internal/dispatch.py:44:0: C0412: Imports from package ctypes are not grouped (ungrouped-imports)
    67: ************* Module skupper_router_internal.management.agent
    67: /home/kgiusti/work/skupper/skupper-router/python/skupper_router_internal/management/agent.py:863:12: E1120: No value for argument 'no_echo' in method call (no-value-for-parameter)
    67: /home/kgiusti/work/skupper/skupper-router/python/skupper_router_internal/management/agent.py:863:12: E1120: No value for argument 'control' in method call (no-value-for-parameter)